offload/liboffload/API/README.md

```diff
@@ -1,24 +1,48 @@
 # Offload API definitions
 
-**Note**: This is a work-in-progress. It is loosely based on equivalent
-tooling in Unified Runtime.
-
 The Tablegen files in this directory are used to define the Offload API. They
 are used with the `offload-tblgen` tool to generate API headers, print headers,
 and other implementation details.
 
 The root file is `OffloadAPI.td` - additional `.td` files can be included in
 this file to add them to the API.
 
+## Modifying the API
+API modifications, including additions, can be made by modifying the existing
+`.td` files. It is also possible to add a new tablegen file to the API by adding
+it to the includes in `OffloadAPI.td`. When Offload is rebuilt the new
+definition will be included in the generated files.
+
+Most API changes and additions do not require any additional work beyond this,
+other than new functions which are described below.
+
+### Adding a new function to the API
+
```

Comment:
This does have a new line before the paragraph, ideally things would be consistent throughout the file.